frontend/public/components/_resource.scss

```diff
@@ -158,3 +158,9 @@
     }
   }
 }
+
+.co-namespace-summary__description {
+  max-height: 15rem;
+  overflow-y: auto;
+  padding-right: var(--pf-global--spacer--sm);
```

Comment:
@debsmita1 Sorry I have still the opinion that we should keep line spaces here. So we could you a markdown component or add just an additional css rule here:

```css
  white-space: pre-wrap;
```

Then also an description like this works:

```yaml
  annotations:
    openshift.io/description: |
      1 first line
      2 second line
            3 line with intent

      4 line break above
      5fasddfasdfafasd
      65faasdfasfdsafasdf
      .....
```

Reply:
I searched for `white-space: pre` in console, there are also a class for that (`co-pre-wrap`). Its up to you, I see other descriptions which uses this also.

Reply:
used the class `co-pre-wrap`